src/views/Pools/components/PoolsTable/Cells/EarningsCell.tsx

```diff
@@ -67,12 +68,20 @@ const EarningsCell: React.FC<EarningsCellProps> = ({ pool, account, userDataLoad
   hasEarnings = isAutoVault ? hasAutoEarnings : hasEarnings
   earningTokenDollarBalance = isAutoVault ? autoUsdToDisplay : earningTokenDollarBalance
 
+  const timeDiffSinceLastAction = (Date.now() - lastActionInMs) / 1000 / 60 / 60
+  const earnedCakePerHour = earningTokenBalance / timeDiffSinceLastAction
```

Comment:
I guess if user didn't stake anything or staked something less than 1 hour ago then`timeDiffSinceLastAction` will be `0` hours. Need to check for `timeDiffSinceLastAction` being `0` and act accordingly (hide this part of the tooltip for example)